src/config.h

```diff
@@ -29,6 +29,11 @@
 #define HAVE_SELECT
 #endif
 
+/*
+ * BOARD_SIZE: Define size of the board to compile Leela with, must be an odd number due to winograd tiles
+ */
+#define BOARD_SIZE 19
+
```

Comment:
#if (BOARD_SIZE % 2 == 0)
#error code assumes odd board size remove at your own risk
#endif

Reply:
Nice, thank you for reviewing the code